portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,52 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.*;
+
+import com.google.gson.Gson;
+
+
+class Comment{
+    private String name;
+    private String content;
+    Comment(String name, String content){
+        this.name = name;
+        this.content = content;
+    }
+} 
+
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-
+  
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("Hello Joey!");
+    // response.setContentType("text/html;");
+    // response.getWriter().println("Hello Joey!");
+    List<Comment> comments = new ArrayList<>();
+    Comment content1 = new Comment("Jack","Hi I am Jack");
+    Comment content2 = new Comment("John","Hi I am John");
+    Comment content3 = new Comment("Jimmy","Hi I am Jimmy");
+    comments.add(content1);
+    comments.add(content2);
+    comments.add(content3);
+    String json = convertToJsonUsingGson(comments);
+
+    // Send the JSON as the response
+    response.setContentType("application/json;");
+    response.getWriter().println(json);
+  }
+
+    /**
```

Comment:
Please follow coding style.

Reply:
Sorry I am not sure what to do about this comment.